cns/restserver/util.go

```diff
@@ -177,7 +177,7 @@ func (service *HTTPRestService) saveNetworkContainerGoalState(
 			fallthrough
 		case cns.AzureFirstParty:
 			fallthrough
-		case cns.WebApps: // todo: Is WebApps an OrchastratorType or ContainerType?
+		case cns.WebApps, cns.BackendNIC: // todo: Is WebApps an OrchastratorType or ContainerType?
```

Comment:
Doesn't this also need to be added to the outer switch statement? If DNC is setting this in the request, it will never reach this line

Reply:
added